chart/values.yaml

```diff
@@ -56,17 +78,13 @@ ingress:
   #    hosts:
   #      - chart-example.local
 
-resources: {}
-  # We usually recommend not to specify default resources and to leave this as a conscious
-  # choice for the user. This also increases chances charts run on environments with little
-  # resources, such as Minikube. If you do want to specify resources, uncomment the following
-  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-  # limits:
-  #   cpu: 100m
-  #   memory: 128Mi
-  # requests:
-  #   cpu: 100m
-  #   memory: 128Mi
+resources: # Resource limits and requests
+  limits:
+    cpu: 500m
```

Comment:
do the limits need to be that high or can we reduce that value?

Reply:
I took the limits from deploy/service.yaml, which have been introduced by @warber 10 months ago:
![image](https://user-images.githubusercontent.com/56065213/146327242-ada965aa-a69b-4e93-aabc-3bf2496cdcba.png)

@warber do the limits have to be that high?